app/views/content/train-to-be-a-teacher/get-school-experience.md

```diff
@@ -13,9 +13,8 @@ calls_to_action:
           Use our service to search for and organise school experience in England.
         link_text: "Search for school experience"
         link_target: "https://schoolexperience.education.gov.uk/"
-cta_mailinglist:
-  mailinglist:
-    text: Get free guidance and support on getting into teaching, including learning more about what teaching is like.
+promo_content:
+  - "content/shared/block-promos/adviser_mailing_find"
```

Comment:
would it be better to have advier-mailing-school here, to reinforce GSE after other content? Sending them to Find seems a bit premature and potentially confusing